README.md

```diff
@@ -219,6 +221,7 @@ Here's the structure visualized as a tree:
 │   ├── azure-ai
 │   ├── azure-dynamic-sessions
 │   ├── langchain-sqlserver
+│   ├── azure-storage
```

Comment:
Let's make the following changes:
* Move `azure-storage` just above the sqlserver package to keep alphabetical order in tact for visualizing the directory tree
* Rename `langchain-sqlserver` to `sqlserver`. The directory name is not correct here so let's just fix it while we are updating it.